STM32F405RGTx/Core/Inc/lut.hpp

```diff
@@ -5,4 +5,3 @@
 // lookup tables
```

Comment:
In general, I think these can be left as `.h`. As a convention, any header files that will work with just C should have `.h`, while header files that can only work with C++ should have `.hpp` imo